sdk/cosmos/azure-cosmos-benchmark/src/main/java/com/azure/cosmos/benchmark/ctl/AsyncCtlWorkload.java

```diff
@@ -88,6 +88,7 @@ public AsyncCtlWorkload(Configuration cfg) {
             .endpoint(cfg.getServiceEndpoint())
             .key(cfg.getMasterKey())
             .consistencyLevel(cfg.getConsistencyLevel())
+            .clientTelemetryEnabled(true)
```

Comment:
something to consider for future (not a blocking question).

can we measure in CTL the perf diff latency P99 and throughput when enabling client telemetry is enabled comparing to disabled.

Reply:
sure